.github/ISSUE_TEMPLATE/survey.md

```diff
@@ -0,0 +1,22 @@
+---
+name: Survey template
+about: Issue for tracking work on OTel surveys. 
+title: 'Survey: <name>'
+labels: survey
+assignees: ''
+
+---
+<!-- Please remember to change the title of this issue – change <name> to the name of your survey. -->
+
+### Tasks
+- [ ] Create a new folder in `sig-end-user/end-user-surveys` named as your survey and add a `README.md` file to it
+- [ ] Prepare questions
+- [ ] Create a questionnaire from [this template](https://docs.google.com/forms/d/1NsOBVcajq3tm4wDrZTt-5bEG5pR3YfK2yccFGVqVzWI/edit) (@avillela can grant you access)
+- [ ] Publish links to questionnaire
```

Comment:
How to get a survey published on website and socials? Who should folks reach out to to get it done?